src/lint/engine/ArcanistLintEngine.php

```diff
@@ -567,6 +567,7 @@ private function beginLintServiceCall(ArcanistLinter $linter, array $paths) {
         'type' => 'lint',
         'linter' => $linter->getInfoName(),
         'paths' => $paths,
+        'linterName' => $linter->getLinterConfigurationName(),
```

Comment:
is the `getLinterConfigurationName` guaranteed to exist?

Reply:
Changed it to getLinterName() since it's abstract at base class so it must be defined at child classes, as for getLinterConfigurationName return null at base class